src/textures/TextTextureRenderer.mjs

```diff
@@ -369,8 +380,8 @@ export default class TextTextureRenderer {
             let spaceLeft = wordWrapWidth;
             let words = lines[i].split(' ');
             for (let j = 0; j < words.length; j++) {
-                let wordWidth = this._context.measureText(words[j]).width;
-                let wordWidthWithSpace = wordWidth + this._context.measureText(' ').width;
+                const wordWidth = this.measureText(words[j], letterSpacing);
+                const wordWidthWithSpace = wordWidth + this._context.measureText(' ').width + letterSpacing;
```

Comment:
You can use `measureText` wrapper you created here.

Reply:
Good call, updated